src/pages/workspace/companyCards/addNew/CardNameStep.tsx

```diff
@@ -0,0 +1,87 @@
+import Text from '@components/Text';
+import TextInput from '@components/TextInput';
+import useAutoFocusInput from '@hooks/useAutoFocusInput';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import * as CompanyCards from '@userActions/CompanyCards';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import INPUT_IDS from '@src/types/form/AddNewCardFeedForm';
+
+function CardNameStep() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const {inputCallbackRef} = useAutoFocusInput();
+    const [addNewCard] = useOnyx(ONYXKEYS.ADD_NEW_COMPANY_CARD);
+
+    const validate = useCallback(
+        (values: FormOnyxValues<typeof ONYXKEYS.FORMS.ADD_NEW_CARD_FEED_FORM>): FormInputErrors<typeof ONYXKEYS.FORMS.ADD_NEW_CARD_FEED_FORM> => {
+            const errors = ValidationUtils.getFieldRequiredErrors(values, [INPUT_IDS.CARD_TITLE]);
+            if (!values.cardTitle) {
+                errors.cardTitle = translate('common.error.fieldRequired');
+            }
+            return errors;
+        },
+        [translate],
+    );
+
+    const submit = useCallback((values: FormOnyxValues<typeof ONYXKEYS.FORMS.ADD_NEW_CARD_FEED_FORM>) => {
+        CompanyCards.setAddNewCompanyCardStepAndData({
+            step: CONST.COMPANY_CARDS.STEP.CARD_DETAILS,
+            data: {
+                cardTitle: values.cardTitle,
+            },
+            isEditing: false,
+        });
+    }, []);
+
+    const handleBackButtonPress = useCallback(() => {
+        CompanyCards.setAddNewCompanyCardStepAndData({step: CONST.COMPANY_CARDS.STEP.CARD_INSTRUCTIONS});
+    }, []);
+
+    return (
+        <ScreenWrapper
+            testID={CardNameStep.displayName}
+            includeSafeAreaPaddingBottom={false}
+            shouldEnablePickerAvoiding={false}
+            shouldEnableMaxHeight
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.companyCards.addCardFeed')}
+                onBackButtonPress={handleBackButtonPress}
+            />
+            <Text style={[styles.textHeadlineLineHeightXXL, styles.ph5, styles.mv3]}>{translate('workspace.companyCards.addNewCard.whatBankIssuesCard')}</Text>
+            <FormProvider
+                formID={ONYXKEYS.FORMS.ADD_NEW_CARD_FEED_FORM}
+                submitButtonText={translate('common.next')}
+                onSubmit={submit}
+                validate={validate}
+                style={[styles.mh5, styles.flexGrow1]}
+                enabledWhenOffline
+            >
+                <InputWrapper
+                    InputComponent={TextInput}
+                    inputID={INPUT_IDS.CARD_TITLE}
+                    label={translate('workspace.companyCards.addNewCard.enterNameOfBank')}
+                    role={CONST.ROLE.PRESENTATION}
+                    // TODO: default value for card name
```

Comment:
We should not leave TODOs in PRs... what should be the default?

Reply:
forgot to remove, thanks